cookbooks/fb_apache/recipes/default.rb

```diff
@@ -81,6 +112,16 @@
   notifies :restart, 'service[apache]'
 end
 
+# Remove unmanaged sites
+::Dir.glob("#{sitesdir}/*").reject {|e| ::File.directory?(e)}.each do |entry|
+  next if ['fb_sites.conf', 'fb_apache.conf'].include?(::File.basename(entry))
```

Comment:
We can't do this if confdir and sitesdir are the same because we'll nuke the ssl.conf and php.conf and stuff that comes with the package and sets some sane defaults... doing this on Debian is clearly a good call though.